src/ServiceControl.Persistence.RavenDB/DatabaseSetup.cs

```diff
@@ -29,16 +29,17 @@ public async Task Execute(CancellationToken cancellationToken)
 
         async Task CreateDatabase(string databaseName, CancellationToken cancellationToken)
         {
-            DatabaseRecordWithEtag dbRecord =
-                await documentStore.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(databaseName),
-                    cancellationToken);
+            var dbRecord = await documentStore.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(databaseName), cancellationToken);
 
             if (dbRecord == null)
             {
                 try
                 {
-                    await documentStore.Maintenance.Server.SendAsync(
-                        new CreateDatabaseOperation(new DatabaseRecord(databaseName)), cancellationToken);
+                    var databaseRecord = new DatabaseRecord(databaseName);
+                    databaseRecord.Settings.Add("Indexing.Auto.SearchEngineType", "Corax");
+                    databaseRecord.Settings.Add("Indexing.Static.SearchEngineType", "Corax");
```

Comment:
Would this affect only brand new databases or would it amend existing databases as well?

Reply:
Only new ones. We need to add something more if we want to update existing databases also.

Reply:
Well, an existing database is going to have all the indexes anyway, unless we were to add one which we're not currently planning, so I guess it's a difference without distinction really.